core-java-modules/core-java-regex/src/test/java/com/baeldung/replacetokens/ReplacingTokensUnitTest.java

```diff
@@ -0,0 +1,80 @@
+package com.baeldung.replacetokens;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.baeldung.replacetokens.ReplacingTokens.TITLE_CASE_PATTERN;
+import static com.baeldung.replacetokens.ReplacingTokens.replaceTokens;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReplacingTokensUnitTest {
+    private static final String EXAMPLE_INPUT = "First 3 Capital Words! then 10 TLAs, I Found";
+    private static final String EXAMPLE_INPUT_PROCESSED = "first 3 capital words! then 10 TLAs, i found";
+
+    @Test
+    public void whenFindMatches_thenTitleWordsFound() {
+        Matcher matcher = TITLE_CASE_PATTERN.matcher(EXAMPLE_INPUT);
+        List<String> matches = new ArrayList<>();
+        while (matcher.find()) {
+            matches.add(matcher.group(1));
+        }
+
+        assertThat(matches)
+                .containsExactly("First", "Capital", "Words", "I", "Found");
```

Comment:
adjust indentation on continuation lines of all the assertThat calls -- two spaces over from where assertThat begins